src/app/components/Item/ItemTable.jsx

```diff
@@ -24,38 +23,9 @@ const ItemTable = ({ items, bibId, getRecord }) => {
       </thead>
       <tbody>
         {
-          items.map((h, i) => {
```

Comment:
When the design is settled,  we need to update the test for this component as the structure has changed.